packages/web-runtime/src/index.js

```diff
@@ -81,7 +82,8 @@ const supportedLanguages = {
   it: 'Italiano',
   gl: 'Galego'
 }
-const translations = coreTranslations
+
+const translations = Object.assign(coreTranslations, odsTranslations)
```

Comment:
object.assign is not deep. For example

```javascript
Object.assign({de: {foo: 1}}, {de: {bar: 1}})
```

will result in 
```javascript
{de: {bar: 1}}
```

and not 
```javascript
{de: {foo: 1, bar: 1}}
```

it's saver to use lodash
```javascript
const foo = merge({}, coreTranslations, odsTranslations)
```